docs/source/reference/fftpack.rst

```diff
@@ -24,4 +24,8 @@ Fast Fourier Transforms
 
 Code compatibility features
 ---------------------------
-The ``get_fft_plan`` function has no counterpart in ``scipy.fftpack``. It returns a cuFFT plan that can be passed to the FFT functions in this module (using the argument ``plan``) to accelarate the computation.
+1. The ``get_fft_plan`` function has no counterpart in ``scipy.fftpack``. It returns a cuFFT plan that can be passed to the FFT functions in this module (using the argument ``plan``) to accelarate the computation.
+
+2. The boolean switch ``cupy.fft.config.enable_nd_planning`` also affects the FFT functions in this module, see :doc:`./fft`. This switch is neglected when planning "manually" using ``get_fft_plan``.
+
+3. Like in ``scipy.fftpack``, all FFT functions in this module have an optional argument ``overwrite_x`` (default is ``False``), which has the same semantics as in ``scipy.fftpack``: when it is set to ``True``, the input array ``x`` *can* (not *will*) be destroyed and replaced by the output. Therefore, to guarantee an in-place FFT is successfully performed, one should always re-assign the input: ``x = cupyx.scipy.fftpack.fft(x, ..., overwrite_x=True, ...)``.
```

Comment:
One nitpick about the last sentence: It is not really an "in-place FFT" if it is necessary to reassign the input. 

Maybe replace the last sentence with:

For this reason, when an in-place FFT is desired, the user should always reassign the input in the following manner: ``x = cupyx.scipy.fftpack.fft(x, ..., overwrite_x=True, ...)``.

Reply:
> One nitpick about the last sentence: It is not really an "in-place FFT" if it is necessary to reassign the input.

Yeah I know...This was precisely I raised #1988...